quetz/db_models.py

```diff
@@ -200,14 +200,32 @@ class Channel(Base):
     )
     description = Column(String)
     private = Column(Boolean, default=False)
-    mirror_channel_url = Column(String)
+    mirror_channel_url = Column("mirror_channel_url", String)
     mirror_mode = Column(String)
     channel_metadata = Column(String, server_default='{}', nullable=False)
     timestamp_mirror_sync = Column(Integer, default=0)
     size = Column(BigInteger, default=0)
     size_limit = Column(BigInteger, default=None)
     ttl = Column(Integer, server_default=f'{60 * 60 * 10}', nullable=False)  # 10 hours
 
+    @property
```

Comment:
I didn't want to change the database type since it would also complicate the schema for such a small change.
Obviously we don't have any concrete numbers but I guess most users would only want to mirror one other channel instead of multiple ones.

Are you in favor of having a mapping to a list of mirror channel urls?